models/training-tuning-scripts/fraud-detection-models/README.md

```diff
@@ -23,13 +23,17 @@ limitations under the License.
 
 Install packages for training GNN model.
 
-```
-mamba env update -n ${CONDA_DEFAULT_ENV} -f requirements.yml
+```bash
+export CUDA_VER=11.8
+mamba install -n base -c conda-forge conda-merge
+conda run -n base --live-stream conda-merge docker/conda/environments/cuda${CUDA_VER}_dev.yml \
+  requirements.yml > .tmp/merged.yml \
+  && mamba env update -n ${CONDA_DEFAULT_ENV} --file .tmp/merged.yml
```

Comment:
needs path to `requirements.yml`:
```
models/training-tuning-scripts/fraud-detection-models/requirements.yml
```

Reply:
I get the following with new `conda-merge` instructions:
<details><summary>Click here to see error details</summary><pre>

Could not solve for environment specs
The following packages are incompatible
├─ boost-cpp 1.82**  is installable with the potential options
│  ├─ boost-cpp 1.82.0 would require
│  │  └─ libboost 1.82.0 ha8e66a6_0, which can be installed;
│  ├─ boost-cpp 1.82.0 would require
│  │  ├─ icu >=73.2,<74.0a0 , which can be installed;
│  │  └─ libboost-devel [1.82.0 h00ab1b0_3|1.82.0 h00ab1b0_4|1.82.0 h00ab1b0_5], which requires
│  │     └─ libboost [1.82.0 h6fcfa73_3|1.82.0 h6fcfa73_4|1.82.0 h6fcfa73_5], which requires
│  │        └─ icu >=73.2,<74.0a0 , which can be installed;
│  ├─ boost-cpp 1.82.0 would require
│  │  └─ icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;
│  ├─ boost-cpp 1.82.0 would require
│  │  └─ libboost <0 , which conflicts with any installable versions previously reported;
│  ├─ boost-cpp 1.82.0 would require
│  │  └─ libboost 1.82.0 ha8e66a6_1, which can be installed;
│  └─ boost-cpp 1.82.0 would require
│     └─ libboost 1.82.0 h109eef0_2, which can be installed;
├─ clangdev 16**  is installable with the potential options
│  ├─ clangdev 16.0.6 would require
│  │  └─ libclang 16.0.6 default_h1cdf331_0, which can be installed;
│  ├─ clangdev 16.0.0 would require
│  │  └─ libclang-cpp 16.0.0 default_h62803fd_0, which requires
│  │     └─ libclang-cpp16 16.0.0 default_h62803fd_0, which can be installed;
│  ├─ clangdev 16.0.0 would require
│  │  └─ libclang-cpp 16.0.0 default_h62803fd_1, which requires
│  │     └─ libclang-cpp16 16.0.0 default_h62803fd_1, which can be installed;
│  ├─ clangdev 16.0.1 would require
│  │  └─ libclang-cpp 16.0.1 default_h62803fd_0, which requires
│  │     └─ libclang-cpp16 16.0.1 default_h62803fd_0, which can be installed;
│  ├─ clangdev 16.0.2 would require
│  │  └─ libclang-cpp 16.0.2 default_h83cc7fd_0, which requires
│  │     └─ libclang-cpp16 16.0.2 default_h83cc7fd_0, which can be installed;
│  ├─ clangdev 16.0.3 would require
│  │  └─ libclang-cpp 16.0.3 default_h1cdf331_2, which requires
│  │     └─ libclang-cpp16 16.0.3 default_h1cdf331_2, which can be installed;
│  ├─ clangdev 16.0.3 would require
│  │  └─ libclang-cpp 16.0.3 default_h83cc7fd_0, which requires
│  │     └─ libclang-cpp16 16.0.3 default_h83cc7fd_0, which can be installed;
│  ├─ clangdev 16.0.3 would require
│  │  └─ libclang-cpp 16.0.3 default_h83cc7fd_1, which requires
│  │     └─ libclang-cpp16 16.0.3 default_h83cc7fd_1, which can be installed;
│  ├─ clangdev 16.0.4 would require
│  │  └─ libclang-cpp 16.0.4 default_h1cdf331_0, which requires
│  │     └─ libclang-cpp16 16.0.4 default_h1cdf331_0, which can be installed;
│  ├─ clangdev 16.0.5 would require
│  │  └─ libclang-cpp 16.0.5 default_h1cdf331_0, which requires
│  │     └─ libclang-cpp16 16.0.5 default_h1cdf331_0, which can be installed;
│  ├─ clangdev 16.0.6 would require
│  │  └─ libclang 16.0.6 default_h1cdf331_1, which can be installed;
│  └─ clangdev 16.0.6 would require
│     └─ libclang 16.0.6 default_h1cdf331_2, which can be installed;
├─ cuml 23.06**  is installable with the potential options
│  ├─ cuml 23.06.00 would require
│  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  └─ cuml 23.06.00 would require
│     ├─ python >=3.10,<3.11.0a0  with the potential options
│     │  ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│     │  ├─ python 3.10.8 would require
│     │  │  └─ graalpy >=23.0.0,<23.0.1.0a0 , which requires
│     │  │     └─ python_abi 3.10.* *_graalpy230_310_native, which can be installed;
│     │  ├─ python [3.10.10|3.10.11|...|3.12.0] would require
│     │  │  └─ libuuid >=1.41.5,<2.0a0 , which can be installed;
│     │  └─ python 3.10.13 would require
│     │     └─ libffi >=3.4,<4.0a0 , which can be installed;
│     └─ python_abi 3.10.* *_cp310 with the potential options
│        ├─ python_abi 3.10 conflicts with any installable versions previously reported;
│        └─ python_abi 3.10 would require
│           └─ python 3.10.* *_cpython, which can be installed;
├─ include-what-you-use 0.20**  is uninstallable because it requires
│  └─ libclang-cpp16 >=16.0.6,<16.1.0a0 , which conflicts with any installable versions previously reported;
├─ libwebp >=1.3.2  is installable with the potential options
│  ├─ libwebp 1.3.2 would require
│  │  └─ jpeg >=9e,<10a , which can be installed;
│  └─ libwebp 1.3.2 would require
│     ├─ libjpeg-turbo [>=2.1.5.1,<3.0a0 |>=3.0.0,<4.0a0 ] but there are no viable options
│     │  ├─ libjpeg-turbo 2.1.5.1 would require
│     │  │  └─ jpeg <0.0.0a , which conflicts with any installable versions previously reported;
│     │  └─ libjpeg-turbo 3.0.0 would require
│     │     └─ jpeg <0.0.0a , which conflicts with any installable versions previously reported;
│     └─ libtiff >=4.6.0,<4.7.0a0 , which requires
│        └─ libjpeg-turbo [>=2.1.5.1,<3.0a0 |>=3.0.0,<4.0a0 ], which cannot be installed (as previously explained);
├─ matplotlib   is installable with the potential options
│  ├─ matplotlib [2.0.2|2.1.0|...|2.2.3] would require
│  │  └─ python >=2.7,<2.8.0a0 , which can be installed;
│  ├─ matplotlib [1.5.3|2.1.2|...|3.1.3] conflicts with any installable versions previously reported;
│  ├─ matplotlib 2.2.5 would require
│  │  └─ matplotlib-base >=2.2.5,<2.2.6.0a0  with the potential options
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2] would require
│  │     │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │     │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │     │  │  └─ python 3.6.12 would require
│  │     │  │     └─ pypy3.6 7.3.3.* , which can be installed;
│  │     │  └─ python_abi 3.6.* *_cp36m, which can be installed;
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2] would require
│  │     │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │     │  └─ python_abi 3.7.* *_cp37m, which can be installed;
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2] would require
│  │     │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │     │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  │     ├─ matplotlib-base [2.2.5|3.3.2] would require
│  │     │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │     │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  │     └─ matplotlib-base 2.2.5 would require
│  │        ├─ python >=2.7,<2.8.0a0 , which can be installed;
│  │        └─ python_abi 2.7.* *_cp27mu, which can be installed;
│  ├─ matplotlib 3.0.1 would require
│  │  └─ matplotlib-base 3.0.1 py37hc2fb212_1001, which does not exist (perhaps a missing channel);
│  ├─ matplotlib 3.0.2 would require
│  │  └─ matplotlib-base 3.0.2 py36h167e16e_1000, which does not exist (perhaps a missing channel);
│  ├─ matplotlib 3.0.2 would require
│  │  └─ matplotlib-base 3.0.2 py36h167e16e_1001, which does not exist (perhaps a missing channel);
│  ├─ matplotlib 3.0.2 would require
│  │  └─ matplotlib-base 3.0.2 py37h167e16e_1001, which does not exist (perhaps a missing channel);
│  ├─ matplotlib 3.2.0 would require
│  │  └─ matplotlib-base >=3.2.0,<3.2.1.0a0  with the potential options
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2] would require
│  │     │  └─ python >=3.6,<3.7.0a0  with the potential options
│  │     │     ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │     │     └─ python 3.6.12, which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2] would require
│  │     │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  │     └─ matplotlib-base [3.1.2|3.1.3|...|3.3.2] would require
│  │        └─ python >=3.8,<3.9.0a0 , which can be installed;
│  ├─ matplotlib 3.2.1 would require
│  │  └─ matplotlib-base >=3.2.1,<3.2.2.0a0 , which can be installed (as previously explained);
│  ├─ matplotlib [3.2.2|3.3.0|3.3.1] would require
│  │  └─ matplotlib-base [>=3.2.2,<3.2.3.0a0 |>=3.3.0,<3.3.1.0a0 |>=3.3.1,<3.3.2.0a0 ] with the potential options
│  │     ├─ matplotlib-base [3.2.2|3.3.0|3.3.1|3.3.2] would require
│  │     │  └─ python_abi 3.6 *_pypy36_pp73, which can be installed;
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  │     └─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  ├─ matplotlib 3.3.2 would require
│  │  └─ matplotlib-base >=3.3.2,<3.3.3.0a0  with the potential options
│  │     ├─ matplotlib-base [3.2.2|3.3.0|3.3.1|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.2.1|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [3.1.2|3.1.3|...|3.3.2], which can be installed (as previously explained);
│  │     ├─ matplotlib-base [2.2.5|3.3.2], which can be installed (as previously explained);
│  │     └─ matplotlib-base 3.3.2 would require
│  │        └─ python >=3.9,<3.10.0a0 , which can be installed;
│  ├─ matplotlib [3.3.2|3.3.3|3.3.4] would require
│  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  └─ python_abi 3.6.* *_cp36m, which can be installed;
│  ├─ matplotlib [3.3.2|3.3.3] would require
│  │  └─ python_abi 3.6 *_pypy36_pp73, which can be installed;
│  ├─ matplotlib [3.3.2|3.3.3|...|3.5.3] would require
│  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  └─ python_abi 3.7.* *_cp37m, which can be installed;
│  ├─ matplotlib [3.3.2|3.3.3|...|3.7.3] would require
│  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  ├─ matplotlib [3.3.2|3.3.3|...|3.8.2] would require
│  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  ├─ matplotlib [3.3.3|3.3.4|...|3.5.2] would require
│  │  └─ python_abi 3.7 *_pypy37_pp73, which can be installed;
│  ├─ matplotlib 3.3.4 would require
│  │  ├─ pypy3.6 >=7.3.3 , which can be installed;
│  │  └─ python_abi 3.6 *_pypy36_pp73, which can be installed;
│  ├─ matplotlib [3.4.3|3.5.0|3.5.2|3.5.3|3.6.0] would require
│  │  ├─ pyqt   with the potential options
│  │  │  ├─ pyqt 4.11.4 would require
│  │  │  │  ├─ python 2.7* , which can be installed;
│  │  │  │  └─ qt [4.8.* |>=4.8.6,<5.0 ], which requires
│  │  │  │     └─ icu 58.*  with the potential options
│  │  │  │        ├─ icu 58.1, which can be installed;
│  │  │  │        └─ icu 58.2 conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt 4.11.4 would require
│  │  │  │  ├─ python 3.4* , which can be installed;
│  │  │  │  └─ qt 4.8.* , which can be installed (as previously explained);
│  │  │  ├─ pyqt 4.11.4 would require
│  │  │  │  ├─ python 3.5* , which can be installed;
│  │  │  │  └─ qt [4.8.* |>=4.8.6,<5.0 ], which can be installed (as previously explained);
│  │  │  ├─ pyqt 4.11.4 would require
│  │  │  │  ├─ python 3.6*  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  ├─ python [3.6.0a3|3.6.0a4|...|3.6.0rc1], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  └─ qt [4.8.* |>=4.8.6,<5.0 ], which can be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ pyqtwebengine [5.12.1 py37he336c9b_5|5.12.1 py37he336c9b_6|5.12.1 py37he336c9b_7|5.12.1 py37he336c9b_8] but there are no viable options
│  │  │  │  │  ├─ pyqtwebengine 5.12.1 would require
│  │  │  │  │  │  └─ qt >=5.12.9,<5.13.0a0  but there are no viable options
│  │  │  │  │  │     ├─ qt 5.12.9 would require
│  │  │  │  │  │     │  ├─ icu >=69.1,<70.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     │  ├─ jpeg >=9e,<10a , which can be installed;
│  │  │  │  │  │     │  └─ libclang >=13.0.1,<14.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     ├─ qt 5.12.9 would require
│  │  │  │  │  │     │  ├─ icu >=67.1,<68.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │  │  │     │  └─ libclang >=10.0.1,<11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     ├─ qt 5.12.9 would require
│  │  │  │  │  │     │  ├─ icu >=67.1,<68.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │  │  │     │  └─ libclang >=11.0.0,<12.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     ├─ qt 5.12.9 would require
│  │  │  │  │  │     │  ├─ icu >=68.1,<69.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │  │  │     │  └─ libclang >=11.0.0,<12.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     ├─ qt 5.12.9 would require
│  │  │  │  │  │     │  ├─ icu >=69.1,<70.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │  │  │     │  └─ libclang 13.* , which conflicts with any installable versions previously reported;
│  │  │  │  │  │     └─ qt 5.12.9 would require
│  │  │  │  │  │        ├─ icu >=68.1,<69.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  │        ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │  │  │        └─ libclang >=11.0.1,<12.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │  │  └─ pyqtwebengine 5.12.1 would require
│  │  │  │  │     ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  │     ├─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  │  │     └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  │  └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=2.7,<2.8.0a0 , which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0  but there are no viable options
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.9, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt 5.12.5 would require
│  │  │  │     │  ├─ icu >=64.2,<65.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ libclang >=9.0.0,<10.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     └─ qt [5.12.5|5.12.6] would require
│  │  │  │        ├─ icu >=67.1,<68.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │        └─ libclang >=10.0.1,<11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ pyqt-impl 5.12.3 py310h1f8e252_8, which requires
│  │  │  │  │  └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  │  └─ pyqtwebengine 5.12.1 py310hfcd6d55_8, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ pyqtwebengine [5.12.1 py36h7ec31b9_6|5.12.1 py36h7ec31b9_7], which cannot be installed (as previously explained);
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  ├─ python_abi 3.6.* *_cp36m, which can be installed;
│  │  │  │  └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  ├─ python_abi 3.6.* *_cp36m, which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ pyqtwebengine [5.12.1 py38h7400c14_6|5.12.1 py38h7400c14_7|5.12.1 py38h7400c14_8], which cannot be installed (as previously explained);
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  │  └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  └─ qt >=5.12.5,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3 would require
│  │  │  │  ├─ pyqtwebengine [5.12.1 py39h0fcd23e_6|5.12.1 py39h0fcd23e_7|5.12.1 py39h0fcd23e_8], which cannot be installed (as previously explained);
│  │  │  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.9.* *_cp39, which can be installed;
│  │  │  │  └─ qt >=5.12.9,<5.13.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9] would require
│  │  │  │  └─ qt-main [>=5.15.3,<5.16.0a0 |>=5.15.4,<5.16.0a0 |>=5.15.6,<5.16.0a0 |>=5.15.8,<5.16.0a0 ] but there are no viable options
│  │  │  │     ├─ qt-main [5.15.3|5.15.4|5.15.6|5.15.8] would require
│  │  │  │     │  └─ icu >=70.1,<71.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qt-main 5.15.8 would require
│  │  │  │     │  ├─ fontconfig >=2.14.2,<3.0a0 , which can be installed;
│  │  │  │     │  ├─ icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ krb5 >=1.20.1,<1.21.0a0 , which can be installed;
│  │  │  │     │  └─ libjpeg-turbo >=2.1.5.1,<3.0a0 , which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8 would require
│  │  │  │     │  ├─ fontconfig >=2.14.2,<3.0a0 , which can be installed;
│  │  │  │     │  ├─ icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ krb5 >=1.20.1,<1.21.0a0 , which can be installed;
│  │  │  │     │  ├─ libclang >=15.0.7,<16.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ libjpeg-turbo >=2.1.5.1,<3.0a0 , which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8 would require
│  │  │  │     │  ├─ fontconfig >=2.14.2,<3.0a0 , which can be installed;
│  │  │  │     │  ├─ icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ libclang >=15.0.7,<16.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     └─ qt-main 5.15.8 would require
│  │  │  │        ├─ icu >=73.2,<74.0a0 , which can be installed;
│  │  │  │        ├─ krb5 >=1.21.2,<1.22.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │        ├─ libclang >=15.0.7,<16.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │        └─ libjpeg-turbo [>=2.1.5.1,<3.0a0 |>=3.0.0,<4.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.4] would require
│  │  │  │  └─ qt-main [5.15.* |>=5.15.2,<5.16.0a0 ] but there are no viable options
│  │  │  │     ├─ qt-main [5.15.3|5.15.4|5.15.6|5.15.8], which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.8, which cannot be installed (as previously explained);
│  │  │  │     ├─ qt-main 5.15.2 would require
│  │  │  │     │  ├─ icu >=69.1,<70.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │     │  └─ libclang 13.* , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qt-main 5.15.2 would require
│  │  │  │     │  ├─ icu >=69.1,<70.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ jpeg >=9e,<10a , which can be installed;
│  │  │  │     │  └─ libclang >=13.0.1,<14.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qt-main 5.15.2 would require
│  │  │  │     │  ├─ icu >=68.2,<69.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ jpeg >=9d,<10a , which can be installed;
│  │  │  │     │  └─ libclang 13.* , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qt-main 5.15.2 would require
│  │  │  │     │  ├─ icu >=73.1,<74.0a0 , which can be installed;
│  │  │  │     │  ├─ jpeg >=9e,<10a , which can be installed;
│  │  │  │     │  ├─ krb5 >=1.20.1,<1.21.0a0 , which can be installed;
│  │  │  │     │  └─ libclang >=14.0.6,<15.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qt-main 5.15.2 would require
│  │  │  │     │  ├─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ jpeg >=9e,<10a , which can be installed;
│  │  │  │     │  └─ libclang >=10.0.1,<11.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     └─ qt-main 5.15.2 would require
│  │  │  │        ├─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │        ├─ jpeg >=9e,<10a , which can be installed;
│  │  │  │        └─ libclang >=14.0.6,<15.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt 5.15.4 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  │  └─ qt-main >=5.15.2,<5.16.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7] would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.7.* *_cp37m, which can be installed;
│  │  │  │  └─ qt-main [>=5.15.3,<5.16.0a0 |>=5.15.4,<5.16.0a0 |>=5.15.6,<5.16.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9] would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  │  └─ qt-main [>=5.15.3,<5.16.0a0 |>=5.15.4,<5.16.0a0 |>=5.15.6,<5.16.0a0 |>=5.15.8,<5.16.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.4 would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.8.* *_cp38, which can be installed;
│  │  │  │  └─ qt-main >=5.15.2,<5.16.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9] would require
│  │  │  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.9.* *_cp39, which can be installed;
│  │  │  │  └─ qt-main [>=5.15.3,<5.16.0a0 |>=5.15.4,<5.16.0a0 |>=5.15.6,<5.16.0a0 |>=5.15.8,<5.16.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.4 would require
│  │  │  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.9.* *_cp39, which can be installed;
│  │  │  │  └─ qt-main >=5.15.2,<5.16.0a0 , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.7|5.15.9] would require
│  │  │  │  ├─ python >=3.11,<3.12.0a0  but there are no viable options
│  │  │  │  │  ├─ python [3.11.0|3.11.1|...|3.11.6] conflicts with any installable versions previously reported;
│  │  │  │  │  └─ python [3.11.0|3.11.2|3.11.3|3.11.4|3.11.5] would require
│  │  │  │  │     └─ libuuid >=1.41.5,<2.0a0 , which can be installed;
│  │  │  │  ├─ python_abi 3.11.* *_cp311, which can be installed;
│  │  │  │  └─ qt-main [>=5.15.6,<5.16.0a0 |>=5.15.8,<5.16.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0 would require
│  │  │  │  ├─ python [2.7* |>=2.7,<2.8.0a0 ], which can be installed;
│  │  │  │  └─ qt [5.6.* |>=5.6.2,<5.7.0a0 ], which requires
│  │  │  │     └─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt 5.6.0 would require
│  │  │  │  ├─ python [3.5* |>=3.5,<3.6.0a0 ], which can be installed;
│  │  │  │  └─ qt [5.6.* |>=5.6.2,<5.7.0a0 |>=5.6.3,<5.7.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0 would require
│  │  │  │  ├─ python 3.6*  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  ├─ python [3.6.0a3|3.6.0a4|...|3.6.0rc1], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  └─ qt 5.6.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0 would require
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  └─ qt [5.6.* |>=5.6.2,<5.7.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  └─ qt [5.6.* |>=5.6.2,<5.7.0a0 ], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  ├─ python >=2.7,<2.8.0a0 , which can be installed;
│  │  │  │  └─ qt 5.9.*  but there are no viable options
│  │  │  │     ├─ qt 5.9.7 would require
│  │  │  │     │  ├─ gstreamer >=1.14.4,<1.15.0a0 , which requires
│  │  │  │     │  │  └─ libffi >=3.2.1,<3.3.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  ├─ icu >=64.2,<65.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ jpeg >=9c,<10a , which can be installed;
│  │  │  │     ├─ qt 5.9.7 would require
│  │  │  │     │  ├─ gstreamer >=1.14.4,<1.15.0a0 , which cannot be installed (as previously explained);
│  │  │  │     │  ├─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ jpeg >=9c,<10a , which can be installed;
│  │  │  │     ├─ qt [5.9.4|5.9.5|5.9.6] would require
│  │  │  │     │  ├─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     │  └─ jpeg >=9b,<10a , which can be installed;
│  │  │  │     └─ qt [5.6.2|5.6.3|5.9.7], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  ├─ python >=3.6,<3.7.0a0  with the potential options
│  │  │  │  │  ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │  │  │  │  └─ python 3.6.12, which can be installed (as previously explained);
│  │  │  │  └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.9 would require
│  │  │  │  ├─ python >=3.12.0rc3,<3.13.0a0  with the potential options
│  │  │  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  │  │  ├─ python 3.12.0 would require
│  │  │  │  │  │  └─ python_abi 3.12.* *_cp312, which conflicts with any installable versions previously reported;
│  │  │  │  │  └─ python 3.12.0rc3 would require
│  │  │  │  │     └─ _python_rc  , which does not exist (perhaps a missing channel);
│  │  │  │  └─ python_abi 3.12.* *_cp312, which conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt [5.15.10|5.15.7] would require
│  │  │  │  ├─ python >=3.11,<3.12.0a0  but there are no viable options
│  │  │  │  │  ├─ python [3.11.0|3.11.1|...|3.11.6] conflicts with any installable versions previously reported;
│  │  │  │  │  └─ python [3.11.0|3.11.2|3.11.3|3.11.4|3.11.5], which cannot be installed (as previously explained);
│  │  │  │  └─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.10 would require
│  │  │  │  ├─ python >=3.12,<3.13.0a0  with the potential options
│  │  │  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  │  │  └─ python 3.12.0, which cannot be installed (as previously explained);
│  │  │  │  └─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.7] would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  └─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.7] would require
│  │  │  │  ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │  │  └─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.7 would require
│  │  │  │  ├─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  │  └─ qtwebkit 5.*  but there are no viable options
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=70.1,<71.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=68.2,<69.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=69.1,<70.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=67.1,<68.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=68.1,<69.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  ├─ libjpeg-turbo >=2.1.5.1,<3.0a0 , which cannot be installed (as previously explained);
│  │  │  │     │  └─ qt-main >=5.15.8,<5.16.0a0 , which cannot be installed (as previously explained);
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ libjpeg-turbo >=3.0.0,<4.0a0 , which cannot be installed (as previously explained);
│  │  │  │     ├─ qtwebkit 5.212 would require
│  │  │  │     │  └─ icu >=58.2,<59.0a0 , which conflicts with any installable versions previously reported;
│  │  │  │     └─ qtwebkit 5.212 would require
│  │  │  │        └─ libwebp >=1.2.0,<1.3.0a0 , which conflicts with any installable versions previously reported;
│  │  │  ├─ pyqt 5.15.7 would require
│  │  │  │  ├─ python >=3.7,<3.8.0a0 , which can be installed;
│  │  │  │  └─ qt-main 5.15.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  ├─ python >=3.5,<3.6.0a0 , which can be installed;
│  │  │  │  └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2 would require
│  │  │  │  ├─ python >=3.8,<3.9.0a0 , which can be installed;
│  │  │  │  └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  │  └─ pyqt 5.9.2 would require
│  │  │     ├─ python >=3.9,<3.10.0a0 , which can be installed;
│  │  │     └─ qt 5.9.* , which cannot be installed (as previously explained);
│  │  ├─ python >=3.10,<3.11.0a0  with the potential options
│  │  │  ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │  │  ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  └─ python 3.10.13, which cannot be installed (as previously explained);
│  │  └─ python_abi 3.10.* *_cp310 with the potential options
│  │     ├─ python_abi 3.10 conflicts with any installable versions previously reported;
│  │     └─ python_abi 3.10, which can be installed (as previously explained);
│  ├─ matplotlib [3.5.0|3.5.1|3.5.2|3.5.3|3.6.1] would require
│  │  ├─ pyqt   with the potential options
│  │  │  ├─ pyqt 4.11.4, which can be installed (as previously explained);
│  │  │  ├─ pyqt 4.11.4, which can be installed (as previously explained);
│  │  │  ├─ pyqt 4.11.4, which can be installed (as previously explained);
│  │  │  ├─ pyqt 4.11.4, which can be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.12.3, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.4], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.4, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.4, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.4|5.15.7|5.15.9], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.4, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.7|5.15.9], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.6.0, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.9, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.7], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.10, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.7], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt [5.15.10|5.15.7], which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.7, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.15.7, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  ├─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  │  └─ pyqt 5.9.2, which cannot be installed (as previously explained);
│  │  └─ python >=3.10,<3.11.0a0  with the potential options
│  │     ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │     ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │     ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │     └─ python 3.10.13, which cannot be installed (as previously explained);
│  ├─ matplotlib [3.5.2|3.5.3|...|3.7.2] would require
│  │  └─ python_abi 3.8 *_pypy38_pp73, which can be installed;
│  ├─ matplotlib [3.5.2|3.5.3|...|3.8.2] would require
│  │  └─ python_abi 3.9 *_pypy39_pp73, which can be installed;
│  ├─ matplotlib [3.6.1|3.6.2|...|3.8.2] would require
│  │  ├─ python >=3.11,<3.12.0a0  but there are no viable options
│  │  │  ├─ python [3.11.0|3.11.1|...|3.11.6] conflicts with any installable versions previously reported;
│  │  │  └─ python [3.11.0|3.11.2|3.11.3|3.11.4|3.11.5], which cannot be installed (as previously explained);
│  │  └─ python_abi 3.11.* *_cp311, which can be installed;
│  ├─ matplotlib 3.6.2 would require
│  │  ├─ pyqt >=5 , which cannot be installed (as previously explained);
│  │  ├─ python >=3.10,<3.11.0a0  with the potential options
│  │  │  ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │  │  ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  └─ python 3.10.13, which cannot be installed (as previously explained);
│  │  └─ python_abi 3.10.* *_cp310 with the potential options
│  │     ├─ python_abi 3.10 conflicts with any installable versions previously reported;
│  │     └─ python_abi 3.10, which can be installed (as previously explained);
│  ├─ matplotlib [3.6.2|3.6.3|...|3.8.0] would require
│  │  ├─ pyqt >=5 , which cannot be installed (as previously explained);
│  │  └─ python >=3.10,<3.11.0a0  with the potential options
│  │     ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │     ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │     ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │     └─ python 3.10.13, which cannot be installed (as previously explained);
│  ├─ matplotlib [3.7.0|3.7.2|3.8.0|3.8.2] would require
│  │  ├─ pyqt >=5.10 , which cannot be installed (as previously explained);
│  │  ├─ python >=3.10,<3.11.0a0  with the potential options
│  │  │  ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │  │  ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  └─ python 3.10.13, which cannot be installed (as previously explained);
│  │  └─ python_abi 3.10.* *_cp310 with the potential options
│  │     ├─ python_abi 3.10 conflicts with any installable versions previously reported;
│  │     └─ python_abi 3.10, which can be installed (as previously explained);
│  ├─ matplotlib [3.7.1|3.7.3|3.8.0|3.8.1] would require
│  │  ├─ pyqt >=5.10 , which cannot be installed (as previously explained);
│  │  └─ python >=3.10,<3.11.0a0  with the potential options
│  │     ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
│  │     ├─ python 3.10.8, which cannot be installed (as previously explained);
│  │     ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │     └─ python 3.10.13, which cannot be installed (as previously explained);
│  ├─ matplotlib [3.8.0|3.8.1|3.8.2] would require
│  │  ├─ python >=3.12,<3.13.0a0  with the potential options
│  │  │  ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│  │  │  └─ python 3.12.0, which cannot be installed (as previously explained);
│  │  └─ python_abi 3.12.* *_cp312, which conflicts with any installable versions previously reported;
│  ├─ matplotlib [2.0.2|2.1.0|...|3.0.0] would require
│  │  └─ python >=3.5,<3.6.0a0 , which can be installed;
│  ├─ matplotlib [2.0.2|2.1.0|...|3.3.4] would require
│  │  ├─ matplotlib-base [3.1.2 py36hef1b27d_1|3.1.3 py36hef1b27d_0], which can be installed (as previously explained);
│  │  └─ python >=3.6,<3.7.0a0  with the potential options
│  │     ├─ python [3.6.0|3.6.1|...|3.6.9], which can be installed;
│  │     └─ python 3.6.12, which can be installed (as previously explained);
│  ├─ matplotlib [2.2.2|2.2.3|...|3.5.3] would require
│  │  ├─ matplotlib-base [3.1.2 py37hef1b27d_1|3.1.3 py37hef1b27d_0], which can be installed (as previously explained);
│  │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│  ├─ matplotlib [3.1.1|3.1.2|...|3.7.2] would require
│  │  ├─ matplotlib-base [3.1.2 py38hef1b27d_1|3.1.3 py38hef1b27d_0], which can be installed (as previously explained);
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  ├─ matplotlib [3.3.4|3.4.2|...|3.8.0] would require
│  │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│  ├─ matplotlib [3.6.2|3.7.1|3.7.2|3.8.0] would require
│  │  └─ python >=3.11,<3.12.0a0  but there are no viable options
│  │     ├─ python [3.11.0|3.11.1|...|3.11.6] conflicts with any installable versions previously reported;
│  │     └─ python [3.11.0|3.11.2|3.11.3|3.11.4|3.11.5], which cannot be installed (as previously explained);
│  └─ matplotlib 3.8.0 would require
│     └─ python >=3.12,<3.13.0a0  with the potential options
│        ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
│        └─ python 3.12.0, which cannot be installed (as previously explained);
└─ python 3.10**  is installable with the potential options
   ├─ python [3.10.0|3.10.1|...|3.10.9] conflicts with any installable versions previously reported;
   ├─ python 3.10.8, which cannot be installed (as previously explained);
   ├─ python [3.10.10|3.10.11|...|3.12.0], which can be installed (as previously explained);
   └─ python 3.10.13, which cannot be installed (as previously explained).

</pre></details>

Update works though with original command.

Reply:
Updated all requirements files so that they include defaults as well, and fixed the path here.

Reply:
Tried again with update that added `defaults` channel and still get same incompatibility errors. Did this in dev container. Original `mamba env update -n ${CONDA_DEFAULT_ENV} -f requirements.yml` works.

Reply:
This should be fixed.

Reply:
Alright, based on our discussion; it looks like matplotlib isn't compatible with libwebp 1.3.2, all the solutions mamba seems to resolve downgrade to 1.3.1, but we can't do that because 1.3.1 has CVE issues.

That said, nothing in fraud-detection-models appears to be using matplotlib (its imported, but not used). I'm going to remove matplotlib from the requirements.yml file here for now, if it needs to go back in later we can handle it then.